docs/media-buy/task-reference/create_media_buy.mdx

```diff
@@ -582,46 +582,357 @@ Invalid format example:
 
 ## Asynchronous Operations
 
-This operation can complete instantly or take days depending on complexity and approval requirements.
+This task can complete instantly or take days depending on complexity and approval requirements. The response includes a `status` field that tells you what happened and what to do next.
 
-### Response Patterns
+| Status | Meaning | Your Action |
+|--------|---------|-------------|
+| `completed` | Done immediately | Process the result |
+| `working` | Processing (~2 min) | Poll frequently or wait for webhook |
+| `submitted` | Long-running (hours/days) | Use webhooks or poll infrequently |
+| `input-required` | Needs your input | Read message, respond with info |
+| `failed` | Error occurred | Handle the error |
```

Comment:
We should include all statuses here or refer to this section https://docs.adcontextprotocol.org/docs/protocols/core-concepts#status-values is probably cleaner